src/components/dashboard/jumbotron/OngoingJumbotron.js

```diff
@@ -39,8 +42,8 @@ export default class CurrentJumbotron extends Component {
 
       return (
         <div>
-          <p>{upcoming.name}</p>
           <p>{upcoming.dscript}</p>
+          <p />
```

Comment:
Is this necessary? Might be for styling...

Reply:
Nope. Accidental. Good-bye weird <p />